libraries/AC_Avoidance/AP_OADijkstra.cpp

```diff
@@ -851,7 +851,7 @@ bool AP_OADijkstra::calc_shortest_path(const Location &origin, const Location &d
         }
         // fail if newest node has invalid distance_from_index
         if ((_short_path_data[nidx].distance_from_idx == OA_DIJKSTRA_POLYGON_SHORTPATH_NOTSET_IDX) ||
-            (_short_path_data[nidx].distance_cm >= FLT_MAX)) {
+            (_short_path_data[nidx].distance_m >= FLT_MAX)) {
```

Comment:
What are the units of FLT_MAX?

Reply:
unitless that the float value limit

Reply:
removed the unit from all the variables in SI units